maxim/logger/fireworks/helpers.py

```
@@ -0,0 +1,171 @@
+    capabilities.
+
+    All methods are static and can be called directly on the class without
+    instantiation.
+    """
+
+    @staticmethod
+    def sync_stream_helper(
+        stream: Iterator[ChatCompletionChunk], 
+        generation: Generation, 
+        trace: Trace, 
+        is_local_trace: bool
```

Ruff (0.12.2)

Boolean-typed positional argument in function definition

(FBT001)